### ISSUE TITLE
usage stats

### ISSUE THREAD

**Issue by Dieterbe** [wger-project/wger#1714]
In order to...

* guide certain "product decisions" (e.g. should we focus on nutrition in the app or on the web page? which functions do people prefer to use on mobile vs web page? is there functionality we can delete because no one uses it?)
* see uptake in users of wger, and which forms of publicity are more successful than others
(note that https://wger.de/en/software/features shows "people using wger" - AFAIK this is just a count on the user table, and may include a majority of users who've stopped using wger a while ago. it would be much more interesting to see "active users" (e.g who have used wger in the last week)

...i would like us to collect basic prometheus metrics from the wger backend, which we can then visualize in grafana.
basically a count of these things happening, broken down by "platform" (web vs mobile)
* meal logged
* ingredient logged
* workout / exercise executed
* body measurement logged
* user logged in
* number of users seen in the last week, and/or in the last 14 days

we can then visualize these quite easily in grafana
